meson.build

```diff
@@ -229,7 +229,6 @@ if get_option('enable-test')
 
   nnstreamer_test_conf.set('ENABLE_ENV_VAR', true)
   nnstreamer_test_conf.set('ENABLE_SYMBOLIC_LINK', false)
-  nnstreamer_test_conf.set('TF_MEM_OPTMZ', false)
```

Comment:
```tf_loadModelFile()``` of ```tensor_filter_tensorflow.c``` uses this. Let's keep it although its default value is false.

Reply:
this was already removed (https://github.com/nnsuite/nnstreamer/pull/1577)

Reply:
Oh.. ok then.